src/Sylius/Bundle/ApiBundle/test/.env

```diff
@@ -0,0 +1,19 @@
+# This file is a "template" of which env vars needs to be defined in your configuration or in an .env file
+# Set variables here that may be different on each deployment target of the app, e.g. development, staging, production.
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
```

Comment:
I'm wondering, if we can't just skip these files and hardcode all these values. It doesn't have to be modified we will use this app only for test purposes